extension/src/extension.ts

```diff
@@ -131,6 +133,8 @@ export class Extension extends Disposable {
       new InternalCommands(outputChannel, ...clis)
     )
 
+    const status = this.dispose.track(new Status(config, ...clis))
```

Comment:
[F] There was a bug in the way that we were creating `Status` it did not contain the `dvcViewer`.

Reply:
I'll add a type in a quick follow-up